modules/core/src/main/java/org/apache/ignite/internal/metrics/Duration.java

```diff
@@ -24,33 +24,33 @@
  *
  * <p>Not thread safe.</p>
  */
-class Duration {
+public class Duration {
```

Comment:
It looks like a time span and not a duration; also, current name clashes with `java.time.Duration` which is not a rare guest. Hence I propose renaming this class to `TimeSpan`